src/BenchmarksDriver/Program.cs

```diff
@@ -71,11 +80,13 @@ public class Program
             {
                 { Scenario.Plaintext, _plaintextJob },
                 { Scenario.Json, _jsonJob },
+                { Scenario.Jil, _jilJob },
```

Comment:
If there are no differences between `_jsonJob` and `_jilJob` (I didn't see any), you can delete `_jilJob` and just use `_jsonJob`.